cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -177,7 +177,9 @@ int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
     LOG(INFO) << logPrefix << "Successfully spilled out " << spilledOut << " bytes.";
     uint64_t total = shrunken + spilledOut;
     DLOG(INFO) << logPrefix << "Successfully reclaimed total " << total << " bytes.";
-    return shrunken + spilledOut;
+    return total;
+  } else {
+    LOG(WARNING) << kSpillStrategy << " was not configured.";
```

Comment:
"Skip spill-to-disk operation since .... was not configured"

Would this look better?

Reply:
ok.